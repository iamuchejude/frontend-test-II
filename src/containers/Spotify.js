import React, { Component } from 'react';
import ArtistInfo from '../components/ArtistInfo';
import RelatedArtists from '../components/RelatedArtists';

class Spotify extends Component {
  constructor(props) {
    super(props);
    this.state = {
      artists: {
        
      },
    };
  }

  render() {
    return (
      <div id="spotify">
        <ArtistInfo />
        <RelatedArtists />
      </div>
    );
  }
}

export default Spotify;
