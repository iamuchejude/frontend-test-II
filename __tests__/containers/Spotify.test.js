import React from 'react';
import { shallow } from 'enzyme';
import Spotify from '../../src/containers/Spotify';

describe('App components', () => {
  it('should render app', () => {
    const wrapper  = shallow(<Spotify />);

    expect(wrapper).toMatchSnapshot()
  })
});
