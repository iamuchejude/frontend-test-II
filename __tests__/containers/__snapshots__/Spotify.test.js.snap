// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App components should render app 1`] = `
<div
  id="spotify"
>
  <ArtistInfo />
  <RelatedArtists />
</div>
`;
